frontend/src/api/k8s/__tests__/events.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { k8sListResourceItems } from '@openshift/dynamic-plugin-sdk-utils';
+import { getNotebookEvents } from '~/api';
+import { EventKind } from '~/k8sTypes';
+import { EventModel } from '~/api/models';
+
+jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({
+  k8sListResourceItems: jest.fn(),
+}));
+const k8sListResourceMock = jest.mocked(k8sListResourceItems<EventKind>);
```

Comment:
could you please update this? 
```suggestion
const k8sListResourceItemsMock = jest.mocked(k8sListResourceItems<EventKind>);
```